pkgs/development/python-modules/pygls/default.nix

```diff
@@ -1,30 +1,29 @@
-{ stdenv, buildPythonPackage, isPy3k, fetchFromGitHub
-, mock, pytest, pytest-asyncio
+{ stdenv
```

Comment:
Instead of `stdenv` consume `lib` directly, you don't need `stdenv` anywhere.

Reply:
Thanks. I've been asked to use `stdenv` instead of just `lib` in previous code reviews, but I prefer using `lib`, so I will update it here.

Reply:
I believe `stdenv.lib` is a copy/paste artifact, but someone more experienced may be able to clarify.

Reply:
It wasn't just an artifact, I was explicitly asked to use `stdenv.lib` instead of just `lib`

Reply:
I updated my commit, but I'll leave this as unresolved if anyone else wants to leave comments.

Reply:
@mweinelt IIRC, it was just a convenience to add lib to stdenv.